buildspec.yml

```diff
@@ -39,7 +39,7 @@ phases:
       - GENERIC_TAG="$FRAMEWORK_VERSION-pytorch-$BUILD_ID"
       - DLC_CPU_TAG="$FRAMEWORK_VERSION-dlc-cpu-$BUILD_ID"
       - DLC_GPU_TAG="$FRAMEWORK_VERSION-dlc-gpu-$BUILD_ID"
-      - DLC_EIA_TAG="$FRAMEWORK_VERSION-dlc-eia-$BUILD_ID"
+      - DLC_EIA_TAG="$EIA_FRAMEWORK_VERSION-dlc-eia-$BUILD_ID"
```

Comment:
This is an unrelated bug fix, right?

Reply:
By the way, is EIA_FRAMEWORK_VERSION actually correct? Later in the review, a comment mentions EIA using version 1.5.1, but this file has the variable set to '1.3.1'.

Reply:
Yes, that's right. It is an unrelated bug fix, but both 1.3.1 and 1.5.1 are untestable right now, since they use `mms` instead of `torchserve`. This toolkit is now built around `torchserve` which makes it necessary to publish a PyTorch DLC image for EIA based on `torchserve`.